API Blueprint Specification.md

```diff
@@ -848,7 +848,15 @@ Defined by an [HTTP request method][httpmethods]:
 
 Defined by an action [name (identifier)](#def-identifier) followed by an [HTTP request method][httpmethods] enclosed in square brackets `[]`.
 
-    # <identifier> [<HTTP request method>]
+    ## <identifier> [<HTTP request method>]
+
+**-- or --**
+
+Defined by an action [name (identifier)](#def-identifier) followed by an [HTTP request method][httpmethods] and [URI template][uritemplate] enclosed in square brackets `[]`. This is also known by the name *Inline Action*.
+
+    # <identifier> [<HTTP request method> <URI template>]
+
+> **NOTE:** The inline action is still considered a part of the resource it is nested under even though it's URI template is different to that of the resource.
```

Comment:
This note does not make much sense. Any and all actions – link transitions – defined under a resource are for that particular resource, regardless of where the transition goes. This is like saying "all the links on this HTML page are part of the page even though the link points elsewhere"